### PR TITLE
Minor punctuation and capitalisation changes

### PR DESCRIPTION
**Pull request by marv-tech**
### Description
The significance of this change is that it updates the punctuation used in this sentence to make it easier for the reader to understand. By removing the sudden full stop and capitalization that separated the sentence into two clauses, the sentence will properly convey its original meaning.
_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [x] **Please list the exact browser and operating system versions you tested**: Chrome browser and MacOS
    
    - 
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by marv-tech** [comment]
@thibaudcolas I made some minor changes here, I would like to know what you think.

File: docs/extending/custom_bulk_actions.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The attributes are as follows:
 
 -   `display_name` - The label that will be displayed on the button in the user interface
 -   `aria_label` - The `aria-label` attribute that will be applied to the button in the user interface
--   `action_type` - A unique identifier for the action. Will be required in the url for bulk actions
+-   `action_type` - A unique identifier for the action that will be required in the url for bulk actions
```
Comment (lb-) [inline docs/extending/custom_bulk_actions.md:33]:
```suggestion
-   `action_type` - A unique identifier for the action, required in the url for bulk actions
```

```suggestion
-   `action_type` - A unique identifier for the action (required in the url for bulk actions)
```

Some other options to both condense this line but also aid readability.

**Review by lb- — CHANGES_REQUESTED** [review]
One suggestion for the first change, other change looks good. 

Please can you also take a moment to update the description of the PR - it just contains the template and no context as to why these changes are important and why you think they could be improved. 

Thanks.

File: docs/extending/custom_bulk_actions.md (hunk 1 above)
Comment (marv-tech) [reply docs/extending/custom_bulk_actions.md:33]: @lb- I have updated the description. What do you think?

File: docs/extending/custom_bulk_actions.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The attributes are as follows:
 
 -   `display_name` - The label that will be displayed on the button in the user interface
 -   `aria_label` - The `aria-label` attribute that will be applied to the button in the user interface
--   `action_type` - A unique identifier for the action. Will be required in the url for bulk actions
+-   `action_type` - A unique identifier for the action (required in the url for bulk actions)
```
Comment (lb-) [inline docs/extending/custom_bulk_actions.md:33]:
```suggestion
-   `action_type` - A unique identifier for the action (required in the URL for bulk actions)
```

Let's update this to use the proper acronym URL while we are at it. I will just commit this suggestion.

**Review by lb- — APPROVED** [review]
Approved - thanks @marv-tech  for submitting these items, and thanks for taking the time to respond to feedback.

**Comment by lb-** [comment]
Thanks @marv-tech  merged - congrats on making your first PR into Wagtail

**Comment by marv-tech** [comment]
> Thanks @marv-tech merged - congrats on making your first PR into Wagtail

Thank you so much @lb-

**Pull request merged**